OpenOversight/app/models/config.py

```diff
@@ -33,6 +33,8 @@ def __init__(self):
         self.WTF_CSRF_ENABLED = True
 
         # Mail Settings
+        # TODO: Remove the default once we are able to update the production .env file
+        self.OO_HELP_EMAIL = os.environ.get("OO_HELP_EMAIL", "info@lucyparsonslabs.com")
```

Comment:
This default should make it so we have valid emails in production.

Reply:
I actually think this should default to `OO_SERVICE_EMAIL` rather than be a required variable. I could see many cases where folks use their service email as their admin email, and making this required is essentially a breaking change for anyone upgrading their own instance (if they don't add the new environment variable). What do you think?

Reply:
The only problem with that in my eyes is that all of our emails say that the `OO_SERVICE_EMAIL` is NOT monitored. That being said, if we make the small change to the bottom of the emails, we'd likely be in the clear. I'm cool with this change. I'll tag @abandoned-prototype, so that they're aware of it.

Reply:
Added!